crates/ty_python_semantic/resources/primer/bad.txt

```diff
@@ -1,24 +1,21 @@
 Tanjun  # hangs
-altair  # cycle panics (try_metaclass_)
 antidote  # hangs / slow
 artigraph  # cycle panics (value_type_)
-colour  # cycle panics (try_metaclass_)
 core  # cycle panics (value_type_)
 cpython  # access to field whilst being initialized, too many cycle iterations
 discord.py  # some kind of hang, only when multi-threaded?
-freqtrade  # cycle panics (try_metaclass_)
-hydpy  # cycle panics (try_metaclass_)
-ibis  # cycle panics (try_metaclass_)
+freqtrade  # hangs
```

Comment:
Does this also hang with my most recent fixpoint PR?

Reply:
Yes :/ Not every time, but often enough (even locally) that I don't think we should add it to `good.txt` yet.